tests/jballerina-unit-test/src/test/resources/test-src/functions/extra-parameters.bal

```diff
@@ -0,0 +1,20 @@
+//  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We use hyphens in module names and underscores in bal file names, shall we fix this?